ArduCopter/motors.pde

```diff
@@ -152,30 +158,6 @@ static bool init_arm_motors(bool arming_from_gcs)
     }
     calc_distance_and_bearing();
 
-    uint32_t gyro_cal_time = 0;
-    if(did_ground_start == false) {
-        uint32_t gyro_cal_begin = millis();
-
-        startup_ground(true);
```

Comment:
They are. They will get set to false quickly anyway (when a non-zero throttle is detected by the land detector). Unclear actually why they were getting set to true in the arming sequence in the first place now that I think about it.